setup.py

```diff
@@ -58,6 +58,7 @@
     'azure-storage-blob>=12.0.0,<13',
     'azure-storage-file-datalake>=12.11.0,<13',
     'azure-identity>=1.13.0',
+    'huggingface_hub>=3.15.4',
```

Comment:
Let's move this to a separate optional dependency group called `'hf'` -- see similar options for `'databricks'` and `'alipan'`, for example